src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,13 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = 'Under fconstruction'; // Set the initial state as "Under construction"
+const categoriesSlice = createSlice({
+  name: 'categories',
+  initialState,
+  reducers: {
+    // reducers specific to categories if needed
+  },
+});
+
```

Comment:
- [ ] Great work creating the **_categorySlice_**. The `initialState` should be equal to an empty array and then you ought to create a reducer called `checkStatus` that returns the string `'Under Construction'` if the `action.payload` is equal to `'Under Construction'`. Kindly implement it this way. 👍 💻

Reply:
Thank you @Gambit142. I'll follow your suggestion.